pvlib/pvsystem.py

```diff
@@ -1046,22 +1048,26 @@ def temperature_model_parameters(self, value):
     @property
     @_unwrap_single_value
     def surface_tilt(self):
-        return tuple(array.surface_tilt for array in self.arrays)
+        # TODO: don't access mount attributes directly?
+        return tuple(array.mount.surface_tilt for array in self.arrays)
```

Comment:
I'm open to changing this. I think the primary motivation for this was preserving the `PVSystem` API for single arrays, though I'm worried I've forgotten some relevant discussion from #1076.. One way forward could be 

```python
if len(self.arrays) == 1:
    warnings.warn(pvlibDeprecationWarning, 'PVSystem.surface_tilt attribute is deprecated. Use PVSystem.array.something()'
else:
    raise IdontKnowTheRightException('PVSystem.surface_tilt not supported for multi-array systems. Use...')
```

and similar for surface azimuth.

Reply:
That was the idea. I think this kind of deprecation + error is a reasonable solution. Maybe an `AttributeError`?